Service/package.json

```diff
@@ -13,19 +13,54 @@
     "react-dev": "webpack -d --watch",
     "start": "sleep 20 && node server/index.js",
     "server-dev": "nodemon server/index.js",
-    "test": "jest"
+    "test": "jest",
+    "csv:airbnb": "node --max-old-space-size=4096 ./seed/csv/generator/airbnb.js",
```

Comment:
This is a ton of scripts. Suggestion - you can throw all of these in a .bash file, and just run that bash script from here. So maybe a "nosqlRoomsSeed.bash" and a "nosqlReservationsSeed.bash", then include all of the respective mongoimport commands inside of those, and just run "bash nosqlRoomsSeed.bash" from your package.json script!